trieste/space.py

```diff
@@ -87,6 +90,15 @@ def __pow__(self: SP, other: int) -> SP:
         tf.debugging.assert_positive(other, message="Exponent must be strictly positive")
         return reduce(operator.mul, [self] * other)
 
+    @property
+    def vectorized_batch_size(self) -> int:
+        return self._vectorized_batch_size
+
+    def set_vectorized_batch_size(self, batch_size: int) -> None:
```

Comment:
You should use a property setter here:

```suggestion
    @vectorized_batch_size.setter
    def vectorized_batch_size(self, batch_size: int) -> None:
```

You can then set it with normal-looking assignment `space.vectorized_batch_size = 3`.